x-team/auto-ui
# Allow attaching failed files again in Portfolio UI

**Release Type:** *Bug Fix*

Fixes https://x-team-internal.atlassian.net/browse/XP-2410

## Description

The reason why a user couldn't attach the same file consecutively more than once is because `input` elements don't trigger its `onChange` event if the `value` doesn't change.

After attaching a file once, the `value` attribute would be set to the path of this file, so trying to attach the same file again wouldn't change it, therefore not triggering the event.

This PR clears the `value` attribute after every file is attached so that the `input` behaves as expected.

## Checklist


## Steps to Test or Reproduce

1. Open the Portfolio UI of any applicant
1. Upload any file in any of the text areas or sections
1. After the file upload completes, delete the file.
1. Now try to upload the file again and it will work (It wasn't before)

See GIF

## Impacted Areas in Application

- The `AttachFile` component

## Screenshots

![2410](https://user-images.githubusercontent.com/7514993/48164944-70427c80-e2ba-11e8-8d9e-08dd1d5c3ced.gif)
